rb/lib/selenium/webdriver/bidi/browsing_context.rb

```diff
@@ -94,6 +94,12 @@ def create(type: nil, context_id: nil)
           result = @bidi.send_cmd('browsingContext.create', type: type.to_s, referenceContext: context_id)
           result['context']
         end
+
+        def set_viewport(context_id: nil, width: nil, height: nil, device_pixel_ratio: nil)
```

Comment:
The defaults for width and height feel weird - Is that part of the w3c spec where they default back to something.

If they're mandatory I'd leave them as mandatory kwargs (But then I can see above mostly everything else is `nil` defaulting).

Reply:
According to the BiDi specification none of them are mandatory: https://w3c.github.io/webdriver-bidi/#command-browsingContext-setViewport

```
browsingContext.SetViewportParameters = {
  context: browsingContext.BrowsingContext,
  ? viewport: browsingContext.Viewport / null,
  ? devicePixelRatio: (float .gt 0.0) / null,
}
```